dsb-integration/src/test/java/com.spectralogic.dsbrowser.integration/tasks/Ds3DeleteFolderTaskTest.java

```
@@ -0,0 +1,122 @@
+    private static final Ds3Client client = Ds3ClientBuilder.fromEnv().withHttps(false).build();
+    private static final Ds3ClientHelpers HELPERS = Ds3ClientHelpers.wrap(client);
+    private static final BuildInfoServiceImpl buildInfoService = new BuildInfoServiceImpl();
+    private static final String TEST_ENV_NAME = "Ds3DeleteFolderTaskTest";
+    private static final String bucketName = "DeleteFolderTaskTestBucket";
+    private static final String folderName = "testFolder/";
+    private static TempStorageIds envStorageIds;
+    private static UUID envDataPolicyId;
+
+    @Before
+    public void setUp() throws InterruptedException {
+        new JFXPanel();
+        final CountDownLatch latch = new CountDownLatch(1);
+        Platform.runLater(() -> {
+            final SavedSession savedSession = new SavedSession(
+                    TEST_ENV_NAME,
+                    client.getConnectionDetails().getEndpoint(),
+                    "80",
+                    null,
+                    new SavedCredentials(
+                            client.getConnectionDetails().getCredentials().getClientId(),
+                            client.getConnectionDetails().getCredentials().getKey()),
+                    false,
+                    false);
+            session = CreateConnectionTask.createConnection(SessionModelService.setSessionModel(savedSession, false), resourceBundle, buildInfoService);
+            try {
+                envDataPolicyId = IntegrationHelpers.setupDataPolicy(TEST_ENV_NAME, false, ChecksumType.Type.MD5, client);
+                envStorageIds = IntegrationHelpers.setup(TEST_ENV_NAME, envDataPolicyId, client);
+                HELPERS.ensureBucketExists(bucketName, envDataPolicyId);
+                latch.countDown();
+            } catch (IOException e) {
+                e.printStackTrace();
```

If this is a legitimate error condition, then let it fall out to the junit runner instead of handling it here, and swallowing the exception.